sdk/confidentialledger/Azure.Security.ConfidentialLedger/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "net",
```

Comment:
Is this a convention, or arbitrary? If convention, could this be derived automatically?

Reply:
We haven't built assumptions of structure into the test-proxy. This path is used as part of the path when storing your recordings.

It's derived automatically, and will have real use. For stuff going into the assets repo, it has real utility. Otherwise 🤷